### DISCUSSION TITLE
Gitbash and Java

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/OOP-using-Java#39] [Announcements]
People who are in need of help in git bash. Can put their queries here.

**Comment by MYethishwar** [comment]
Mam Why do we need 3 branches in our repository if we're not  dealing with a single complex project?

**Reply by mssanamateen** [reply]
Branches help us to work on different features parallely. Before we merge changes into main branch. It is best to do so in separate branch
